app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/UserType.scala

```diff
@@ -21,16 +21,19 @@ sealed trait UserType
 object UserType {
 
   final case class Individual(
-                               nino: NINO,
+                               id: Either[SAUTR,NINO],
                                name: Name,
-                               dateOfBirth: DateOfBirth,
+                               dateOfBirth: Option[DateOfBirth],
                                email: Option[Email]
                              ) extends UserType
 
   final case class Trust(sautr: SAUTR) extends UserType
 
   final case object OrganisationUnregisteredTrust extends UserType
 
-  final case class InsufficientConfidenceLevel(nino: Option[NINO]) extends UserType
+  final case class InsufficientConfidenceLevel(nino: Option[NINO],
+                                               name: Name,
```

Comment:
Could we rename "InsufficientConfidenceLevel" to "UserWithInsufficientConfidenceLevel" or "EntityWithInsufficientConfidenceLevel"?

Reply:
I renamed it to "IndividualWithInsufficientConfidenceLevel" - confidence level only really applies to individuals